### ISSUE TITLE
missing sidebar 

### ISSUE THREAD

**Issue by boredtan** [tachiyomiorg/tachiyomi#9536]
### Steps to reproduce

Scrolling up and down in library 

### Expected behavior

Sidebar should appear on the right side when scrolling 

### Actual behavior

Sidebar doesn't appear when scrolling.

### Tachiyomi version

0.14.6

### Android version

android 13

### Device

samsung a54 and samsung s10+

### Other details

Sidebar appeared before updating the app. However after updating the app it does not appear anymore. 

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose).
- [X] I have gone through the [FAQ](https://tachiyomi.org/help/faq/) and [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] I have updated the app to version **[0.14.6](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I will fill out all of the requested information in this form.

**Comment by arkon** [comment]
Are you talking about the scrollbar?

**Comment by boredtan** [comment]
Yes. I posted a video in the discord channel about this issue as well a while ago if u need a reference on how it looks like for me.

**Comment by arkon** [comment]
Can you link to your videos? I'm not really interested in digging through all of the messages in #support.

**Comment by boredtan** [comment]
https://discord.com/channels/349436576037732353/349436576037732355/1092856056658526368

Sorry for the late reply. Didn't receive a notification for it.
